node-task-pkg/index.js

```diff
@@ -0,0 +1,51 @@
+const util = require("util");
+const { exec } = require("child_process");
+const execProm = util.promisify(exec);
+
+const os = require("os");
+const plat = os.platform();
+
+exports.getProcList = async function (username) {
+    let result = {stdout:"",stderr:""};
+    try{
+        if (plat == "win32"){
```

Comment:
I think we don't have win32 and others (if and else). We have 3 platforms. Maybe switch will be better? And default case: error because we don't work with this OS

Reply:
Sorry I am confused what you mean, what platforms will we be using? When I request the platform version it is returned like this:
- Windows : win32
- Linux : linux
- Mac OS: darwin

Do we need to support other platforms except for these?

Reply:
A good practice is to handle unsupported os. Write error instead of doing nothing.

`os.platform() `can return 'aix', 'darwin', 'freebsd','linux', 'openbsd', 'sunos', and 'win32'
https://nodejs.org/api/os.html#osplatform

Reply:
Now we only need support for win32, linux, darwin